welder-web.spec

```diff
@@ -0,0 +1,25 @@
+Name: welder-web
+Version: 0
+Release: 0
```

Comment:
LGTM but I'd like to have the version read from the same place where the web code reads it, not duplicate the information inside the spec file.